src/prescriptions/prescription-details.component.tsx

```diff
@@ -26,6 +27,25 @@ const PrescriptionDetails: React.FC<{
     }
   }, [totalAllergies]);
 
+  const generateStatusTag: Function = (
+    medicationRequest: MedicationRequest
+  ) => {
+    if (!medicationRequest.status || medicationRequest.status === "active") {
+      return null;
+    }
+
+    if (medicationRequest.status === "stopped") {
+      return <Tag type="red">{t("expired", "Expired")}</Tag>;
+    }
+
+    if (medicationRequest.status === "cancelled") {
+      return <Tag type="red">{t("cancelled", "Cancelled")}</Tag>;
+    }
+    // TODO support completed status & will need to change expired once we change the definition of expired
+    // TODO will need to support potential Medication Dispense statuses or refactor into different request and dispense components
+    return null;
+  };
```

Comment:
I pulled this out of a separate tag into this details component because it is now the only place where it is being used.  It remains to be seen if we'll want to pull this out again.